operator-common/src/main/java/io/strimzi/operator/common/model/Ca.java

```diff
@@ -1157,4 +1157,18 @@ public boolean hasCaCertGenerationChanged(Secret secret) {
         }
         return false;
     }
+
+
+    /**
+     * Generates the expiration date as epoch of the CA certificate.
+     * @return  Epoch representation of the expiration date of the certificate or 0 if certSecret contains no valid X509Certificate
```

Comment:
Looks like call to cert() raises a RE if the certificate is not valid. This means zero is never returned.

Reply:
Thanks, changing now